plugins/CoreAdminHome/templates/trackingCodeGenerator.twig

```diff
@@ -28,6 +28,7 @@
             <a href="#/wordpress">{{ 'SitesManager_SiteWithoutDataWordpress'|translate }}</a>
             <a href="#/cloudflare">{{ 'SitesManager_SiteWithoutDataCloudflare'|translate }}</a>
             {{ postEvent('Template.endTrackingCodePageTableOfContents') }}
+            <a href="#/vue">{{ 'SitesManager_SiteWithoutDataVue'|translate }}</a>
```

Comment:
@AltamashShaikh should the event be after all the options like so?
```suggestion
            <a href="#/vue">{{ 'SitesManager_SiteWithoutDataVue'|translate }}</a>
            {{ postEvent('Template.endTrackingCodePageTableOfContents') }}            
```

Reply:
Updated as suggested